### ISSUE TITLE
How about the performances of the parser?

### ISSUE THREAD

**Issue by zhangmeishan** [chantera/biaffineparser#3]
Chinese & English.  can  you reproduce the results of the corresponding paper?  thanks very much

**Comment by chantera** [comment]
The model achieved UAS: 94.91 and LAS: 92.47 in wsj_22 (PTB-SD 3.3.0) in the first experimental implementation.
After fixing some detailed settings, it performed UAS 95.4 in wsj_23.
But I don't remember whether I reflected those changes in this repository.

The reason I implemented this model was to investigate the performance of biaffine transformation.
So I did not reproduce the results reported in the paper and will not maintain this repository.

FYI, if you want to do experiments with the exact same settings of the paper, you should make mini-batch with the fixed size (5000 tokens) because the original programs use TensorFlow.
They grouped sentences into 40 buckets using k-mean by the length.
In the training iteration, sentences in the buckets are shuffled, but sentences in different buckets would never be in the same bucket.
Furthermore, the order of buckets would not change.
In my opinion, this mini-batch training affects performance, but I have not investigated it yet.
I think https://github.com/jcyk/Dynet-Biaffine-dependency-parser is more helpful to you.

**Comment by zhangmeishan** [comment]
Thanks.  I actually mean the code of pytorch backend, since I see your results of the chainer model.  As there are several differences for example the dropout, I want to check whether I can obtain a comparable performance by using pytorch without huge efforts. 
I have some experiences from others that the 40-bucket tricky is not so necessary for performance.
I am re-implementing the biaffine parser with pytorch, but it seems very worse, so I want to know some key points which influence the performance much :)
